sklearn/utils/tests/test_multiclass.py

```diff
@@ -290,6 +290,14 @@ def test_type_of_target():
                ' use a binary array or sparse matrix instead.')
         assert_raises_regex(ValueError, msg, type_of_target, example)
 
+    try:
+        from pandas import SparseSeries
+    except ImportError:
+        raise SkipTest("Pandas not found")
+    
+    y = SparseSeries([1, 0, 0, 1, 0])
+    msg = "y cannot be class 'SparseSeries'."
+    assert_raises_regex(ValueError, msg, type_of_target, y)
 
```

Comment:
Per @jnothman 's request, I've only wrapped the import and added `raise SkipTest("Pandas not found")` otherwise.